Services/Dashboard/Block/classes/class.ilSelectedItemsBlockGUI.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+/**
+ * @ilCtrl_IsCalledBy ilSelectedItemsBlockGUI: ilColumnGUI
+ */
+class ilSelectedItemsBlockGUI extends ilDashboardBlockGUI
+{
+    private ilFavouritesManager $favourites;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->favourites = new ilFavouritesManager();
+    }
+
+    public function initViewSettings(): void
+    {
+        $this->viewSettings = new ilPDSelectedItemsBlockViewSettings(
+            $this->user,
+            ilPDSelectedItemsBlockConstants::VIEW_SELECTED_ITEMS
+        );
+
+        $this->ctrl->setParameter($this, 'view', $this->viewSettings->getCurrentView());
+    }
+
+    public function emptyHandling(): string
+    {
+        $this->lng->loadLanguageModule('rep');
+        $txt = $this->lng->txt("rep_fav_intro1") . "<br>";
```

Comment:
You can also change the language vars directly, if you are the only one using them.
Language variables support minor html like `<br>` and stuff.